components/BottomNavigation/src/MDCBottomNavigationBar.m

```diff
@@ -473,7 +473,11 @@ - (void)setItems:(NSArray<UITabBarItem *> *)items {
     [self.itemViews addObject:itemView];
     [self.containerView addSubview:itemView];
   }
+  self.selectedItem = self.items[0];
   [self addObserversToTabBarItems];
+  self.selectedItem = NULL;
+  [self setNeedsLayout];
+  [self setNeedsDisplay];
```

Comment:
Is this required? I don't see a `-drawRect` and I would expect `layoutSubviews` to be sufficient for updating the view.